configs/avae-test-config.yml

```diff
@@ -3,6 +3,8 @@
 datapath: data/shrec21_full_dataset/subtomo_files
 affinity: data/shrec21_full_dataset/subtomo_files/affinity.csv
 classes: data/shrec21_full_dataset/subtomo_files/classes.csv
+state : /your_path_to_saved_state/state.pt
+restart: True
 limit: 1000
 split: 20
```

Comment:
Add gaussian_blur, normalise and shift_mean to default config

Reply:
fixed in [c061edf](https://github.com/alan-turing-institute/affinity-vae/pull/144/commits/c061edf05f2e7f9114d7cc1a7100b2ec826dd691)